src/App.js

```diff
@@ -47,6 +55,13 @@ class App extends Component {
       </div>
     );
   }
+
+  handleAuthenticationInfoUpdate = function(authResponse) {
+    if(authResponse != undefined) {
+      console.log(authResponse);
+      this.setState({authenticationInfo: {access_token: authResponse.access_token, expires_in: authResponse.expires_in}})
+    }
+  }
```

Comment:
else what?
If it is undefined (which probably will never happen), log it. Or throw a custom error.

Reply:
Done?